### ISSUE TITLE
fabric ?

### ISSUE THREAD

**Issue by ydainna** [dscalzi/HeliosLauncher#198]
Hello
is it possible to make the launcher compatible with fabric please?
thanks to you

**Comment by Neerfix** [comment]
see : https://github.com/dscalzi/HeliosLauncher/issues/65

**Comment by MJRamon** [comment]
Gonna raise this question once again, cause it's been a long time without any news on this. Very much needed.

**Comment by dscalzi** [comment]
It's not planned

**Comment by MJRamon** [comment]
Can I ask you to reconsider? Nowadays Fabric is represented widely, and plays a huge role in Minecraft modding world.
Your launcher is one the best out there, and it would be a great tool for all of us, who use Fabric. Please, don't leave us without this opportunity :)

**Comment by dscalzi** [comment]
Once assetguard-2 (#270) lands, the next thing to rework would be processbuilder.js. That would be done with fabric support in mind.

**Comment by dscalzi** [comment]
closing as duplicate of #65
